specs/agents/tracing-instrumentation-db.md

```diff
@@ -117,11 +117,20 @@ The following fields are relevant for database and datastore spans. Where possib
 |`action`| `request` |
 | __**context.db._**__  |<hr/>|<hr/>|
 |`_.instance`| e.g. `my-cluster` | [Cluster name](#cluster-name), if available.
-|`_.statement`| e.g. <pre lang="json">{"query": {"match": {"user.id": "kimchy"}}}</pre> | For Elasticsearch search-type queries, the request body may be recorded. Alternatively, if a query is specified in HTTP query parameters, that may be used instead. If the body is gzip-encoded, the body should be decoded first.|
+|`_.statement`| e.g. <pre lang="json">{"query": {"match": {"user.id": "kimchy"}}}</pre> | For Elasticsearch search-type queries, the request body SHOULD be recorded according to the `elasticsearch_capture_body_urls` option (see below). If the body is gzip-encoded, the body should be decoded first.|
 |`_.type`|`elasticsearch`|
 |`_.user`| :heavy_minus_sign: |
 |`_.link`| :heavy_minus_sign: |
 |`_.rows_affected`| :heavy_minus_sign: |
+| __**context.http._**__ |<hr/>|<hr/>|
+|`_.status_code`|`200`|
+|`_.method`|`GET`|
+|`_.url.full`|`https://localhost:9200/index/_search?q=user.id:kimchy`|
+|`_.url.protocol`|`https`|
+|`_.url.hostname`|`localhost`|
+|`_.url.port`|`9200`|
+|`_.url.pathname`|`/index/_search`|
+|`_.url.search`|`q=user.id:kimchy`|
```

Comment:
These are transaction intake fields. For spans it is just `context.http.url` as a string.

```suggestion
|`_.url`|`https://localhost:9200/index/_search?q=user.id:kimchy`|
```

Reply:
Ah, right, good catch.